cmake/modules/OpenSSL.cmake

```diff
@@ -14,6 +14,7 @@ else()
   set(OPENSSL_LIBRARY_SSL "${OPENSSL_INSTALL_DIR}/lib/libssl.a")
   set(OPENSSL_LIBRARY_CRYPTO "${OPENSSL_INSTALL_DIR}/lib/libcrypto.a")
   set(OPENSSL_BINARY "${OPENSSL_INSTALL_DIR}/bin/openssl")
+  set(USE_BUNDLED_OPENSSL ON)
```

Comment:
Does this needs to be always ON?

We enable it only when USE_BUNDLED_DEPS is ON, for cmake/modules/sysdig.cmake dependency.